api/swagger.yaml

```
@@ -3357,6 +3357,23 @@ definitions:
             format: "dateTime"
           Message:
             type: "string"
+      ServiceStatus:
+        description: |
+          The status of the service's tasks. Provided only when requested as
+          part of a ServiceList operation.
+        type: "object"
+        properties:
+          RunningTasks:
+            description: "The number of tasks for the service currently in the Running state"
+            type: "number"
```

Can you add an example value here, and set the correct format/type?

```suggestion
            type: "integer"
            format: "uint64"
            example: 7
```